test/panels_constants.tsx

```diff
@@ -4,7 +4,7 @@
  */
 
 export const panelBreadCrumbs = [
-  { text: 'Operational panels', href: '#/operational_panels/' },
+  { text: 'Observability Dashboards', href: '#/operational_panels/' },
```

Comment:
should we change the href too?

Reply:
Yes but may not in this PR as this only deals with renaming. Verified in unit test run that there were no failing tests.